helpers/authentication.test.js

```diff
@@ -0,0 +1,82 @@
+import { authTokenKey, clearToken, getToken, setToken } from './authentication';
+
+describe('Authentication helper', () => {
+  const mockStorage = {};
+  const mockLocalStorageGetItem = key => {
+    return mockStorage[key];
+  };
+  const mockLocalStorageSetItem = (key, value) => {
+    mockStorage[key] = value;
+  };
+  const mockLocalStorageRemoveItem = key => {
+    mockStorage[key] = undefined;
+  };
+
+  const mockTokenType = 'token type';
+  const mockAccessToken = 'access token';
+  const mockRefreshToken = 'refresh token';
+
+  beforeEach(() => {
+    window.Storage.prototype.getItem = mockLocalStorageGetItem;
+    window.Storage.prototype.setItem = mockLocalStorageSetItem;
+    window.Storage.prototype.removeItem = mockLocalStorageRemoveItem;
+  });
+
+  describe('getToken', () => {
+    beforeEach(() => {
+      const mockTokens = {
```

Comment:
```suggestion
      const mockToken = {
```
If you agree with the previous suggestion ;-)